examples/svelte-state-rune/src/entrypoints/popup/main.ts

```diff
@@ -0,0 +1,10 @@
+import { mount } from "svelte";
+
+import "./app.css";
+import App from "./app.svelte";
```

Comment:
```suggestion
import App from "./App.svelte";
```

ty for your work btw :)